AudioLink/Scripts/AudioLinkController.cs

```diff
@@ -124,14 +154,6 @@ public void UpdateSettings()
                 audioLink.SetProgramVariable("threshold1", threshold1Slider.value);
                 audioLink.SetProgramVariable("threshold2", threshold2Slider.value);
                 audioLink.SetProgramVariable("threshold3", threshold3Slider.value);
-                audioSpectrumDisplay.SetFloat("_X0", x0Slider.value);
-                audioSpectrumDisplay.SetFloat("_X1", x1Slider.value);
-                audioSpectrumDisplay.SetFloat("_X2", x2Slider.value);
-                audioSpectrumDisplay.SetFloat("_X3", x3Slider.value);
-                audioSpectrumDisplay.SetFloat("_Threshold0", threshold0Slider.value);
-                audioSpectrumDisplay.SetFloat("_Threshold1", threshold1Slider.value);
-                audioSpectrumDisplay.SetFloat("_Threshold2", threshold2Slider.value);
-                audioSpectrumDisplay.SetFloat("_Threshold3", threshold3Slider.value);
```

Comment:
I don't mind at all, but I'm just curious. Why'd you move this?

Reply:
because it does not depend on `audioLink`, which is now gated behind a null-check